src/librustdoc/html/static/storage.js

```diff
@@ -118,7 +118,8 @@ function switchTheme(styleElem, mainStyleElem, newTheme, saveTheme) {
 }
 
 function getSystemValue() {
-    return getComputedStyle(document.documentElement).getPropertyValue('content');
+    var property = getComputedStyle(document.documentElement).getPropertyValue('content');
+    return property.replace(/\"/g, "").replace(/\'/g, "");
```

Comment:
```suggestion
    return property.replace(/[\"\']/g, "");
```

Reply:
In case we change the css for whatever reason, I'd prefer to keep the replace of single quotes as well. Let's avoid surprising regressions if possible.

Reply:
Surely @bjorn3's suggestion replaces both double and single quotes, just in one .replace() call?

Reply:
Oh indeed! Completely missed it... Sorry @bjorn3...